Library/RequestResponseHeaders.ts

```diff
@@ -0,0 +1,23 @@
+export = {
+    /**
+     * Source instrumentation header that is added by an application while making http
+     * requests and retrieved by the other application when processing incoming requests.
+     */
+    sourceInstrumentationKeyHeader: "x-ms-request-source-ikey",
+
+    /**
+     * Target instrumentation header that is added to the response and retrieved by the
+     * calling application when processing incoming responses.
+     */
+    targetInstrumentationKeyHeader: "x-ms-request-target-ikey",
+
+    /**
+     * Standard parent Id header.
+     */
+    standardParentIdHeader: "x-ms-request-id",
+
+    /**
+     * Standard root id header.
+     */
+    standardRootIdHeader: "x-ms-request-root-id",
```

Comment:
I'm not, yet. I copied/translated all these constants from the C# code. I assumed these additional constants might be used later when we add per-request correlation.